.storybook/preview-head.html

```diff
@@ -1,79 +1,98 @@
-
-        /* SPACING */
-        --spacing-super-small: 1px;
-        --spacing-extra-small: 3px;
-        --spacing-small: 6px;
-        --spacing-small-medium: 8px;
-        --spacing-medium: 10px;
-        --spacing-large-medium: 13px;
-        --spacing-large: 16px;
-        --spacing-extra-large: 24px;
-
-        /* DIMENSIONS */
-        --width-separator-thin: 1px;
-        --width-separator-medium: 2px;
-
-        /* FONT SIZE */
-        --font-size-extra-small: 11px;
-        --font-size-small: 12px;
-        --font-size-small-medium: 13px;
-        --font-size-medium: 14px;
-        --font-size-large-medium: 16px;
-        --font-size-large: 18px;
-        --font-size-extra-large: 24px;
-        --font-size-super-large: 32px;
-        --font-size-ultra-large: 48px;
-
-        /* FONT WEIGHT */
-        --font-weight-light: 400;
-        --font-weight-medium: 400;
-        --font-weight-bold: 700;
-
-        /* COLOR PALETTE */
-        --color-primary: #05367d;
-        --color-accent: #289f8c;
-        --color-danger: firebrick;
-        --color-success: forestgreen;
-        --color-warning: tomato;
-
-        /* COLOR GENERAL */
-        --color-foreground: #fff;
-        --color-background: #f0f3f7;
-
-        /* COLOR TEXT */
-        --color-text: #243244;
-        --color-text-on-primary: rgba(255, 255, 255, .9);
-        --color-text-on-accent: rgba(255, 255, 255, .9);
-        --color-text-on-danger: rgba(255, 255, 255, .9);
-        --color-text-on-success: rgba(255, 255, 255, .9);
-        --color-text-on-warning: rgba(255, 255, 255, .9);
-
-        --color-text-label: #576374;
-        --color-text-disabled: #b8c2cd;
-        --color-separator: #b8c2cd;
-
-        --color-button-background: #dae0e7;
-        --color-button-background-disabled: #e4e9ef;
-
-        --radius-button: 4px;
-    }
+:root {
+    --tui-font-family-sans-serif: 'PT Sans', sans-serif;
+    --tui-font-family-monospace: 'Inconsolata', monospace;
+
+    /* SPACING */
+    --tui-spacing-super-small: 1px;
+    --tui-spacing-extra-small: 3px;
+    --tui-spacing-small: 6px;
+    --tui-spacing-small-medium: 8px;
+    --tui-spacing-medium: 10px;
+    --tui-spacing-large-medium: 13px;
+    --tui-spacing-large: 16px;
+    --tui-spacing-extra-large: 24px;
+
+    /* DIMENSIONS */
+    --tui-width-separator-thin: 1px;
+    --tui-width-separator-medium: 2px;
+
+    /* FONT SIZE */
+    --tui-font-size-extra-small: 11px;
+    --tui-font-size-small: 12px;
+    --tui-font-size-small-medium: 13px;
+    --tui-font-size-medium: 14px;
+    --tui-font-size-large-medium: 16px;
+    --tui-font-size-large: 18px;
+    --tui-font-size-extra-large: 24px;
+    --tui-font-size-super-large: 32px;
+    --tui-font-size-ultra-large: 48px;
+
+    /* FONT WEIGHT */
+    --tui-font-weight-light: 400;
+    --tui-font-weight-medium: 400;
+    --tui-font-weight-bold: 700;
+
+    /* COLOR PALETTE */
+    --tui-color-primary: #05367d;
+    --tui-color-accent: #289f8c;
```

Comment:
@frozenhelium lets change this to some other color.
We've been playing with the same teal for quite some time now.

Reply:
@AdityaKhatri "playing with the same teal for quite some time now" is not the good enough reason to change it. It's hard to find the color which suits for both the dark and light background and is easier for testing.

Reply:
@frozenhelium , I am suggesting this as a lot of projects where we use the components are inherently teal.
We do need to make sure it works with other colors as well, as this is made to be used for varieties of projects.

Seconded on making this a separate issue, but definitely should be taken care of.

Reply:
@AdityaKhatri Different project that use this library should use their own primary, secondary colors and other variables. Since this library is supposed to work with both light and dark background simultaneously, it's critical to test out this feature. So if we use new accent color it must provide enough contrast in both light and dark mode